### PR TITLE
web: update landing screenshots

### PR DESCRIPTION
**Pull request by dcposch**
## Web

landing-screenshots

## Readme, email

email

### CODE REVIEW

File: apps/daimo-web/public/assets/landing-screenshots-1.png
Comment (nalinbhardwaj) [inline apps/daimo-web/public/assets/landing-screenshots-1.png:1]: this pic is not the same as the one in the PR description? intentional?

**Comment by nalinbhardwaj** [comment]
Want to update the README as well in this PR?

**Review by nalinbhardwaj — APPROVED** [review]

**Comment by nalinbhardwaj** [comment]
we should upscale these also, just looked at the website on desktop and they look a bit blurry

**Comment by dcposch** [comment]
## Upscaled, web

landing-screenshots

## Upscaled, README and email
ds

**Pull request merged**